docs/operators/kafkaInputOperator.md

```diff
@@ -280,3 +282,147 @@ Below is the configuration for “test” Kafka topic name and
 <value>localhost:2181</value>
 </property>
 ```
+
+
+### 0.9 Version of Kafka Input Operator
+
+### AbstractKafkaInputOperator
+
+This is the abstract implementation which is using 0.9 new consumer API that serves as base class
+for consuming messages from Kafka messaging system. This operator has key features like fault-tolerant, scalable, multi-cluster support, multi-topic support.
```

Comment:
Rewrite both sentences something like this:
This version uses the new 0.9 version of the Consumer API; differences between the two API versions are described here <provide a suitable link>.
This operator is fault-tolerant, scalable, and has multi-cluster and multi-topic support.